### PR TITLE
MGMT-4990 - fix pep8 linter issues

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

File: discovery-infra/test_infra/assisted_service_api.py [hunk 1]
```diff
@@ -174,13 +174,13 @@ def _download(cls, response: HTTPResponse, file_path: str, verify_file_size=Fals
...
-            image_type: str = consts.ImageType.FULL_ISO,
-            static_network_config: Optional[list] = None,
+        self,
+        cluster_id: str,
+        ssh_key: str,
+        image_type: str = consts.ImageType.FULL_ISO,
+        static_network_config: Optional[list] = None,
     ) -> models.cluster.Cluster:
-        cluster = self.cluster_get(cluster_id=cluster_id)
+        self.cluster_get(cluster_id=cluster_id)
```
Comment (osherdp) [inline discovery-infra/test_infra/assisted_service_api.py:183]: then why do we do that?
did we wanted to return it at the end?

File: discovery-infra/test_infra/controllers/nat_controller.py [hunk 2]
```diff
@@ -12,13 +12,14 @@ class NatController:
...
     """
+
     def __init__(self, input_interfaces: Union[List, Tuple], ns_index: Union[str, int]):
         self._input_interfaces = input_interfaces
         self._ns_index = ns_index
         self._mark = self._build_mark()
 
     def add_nat_rules(self) -> None:
-        """" Add rules for the input interfaces and output interfaces """
+        """ " Add rules for the input interfaces and output interfaces"""
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/nat_controller.py:22]: just remove one quote

File: discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py [hunk 3]
```diff
@@ -533,35 +537,32 @@ def set_per_device_boot_order(self, node_name, key: Callable[[Disk], int]):
...
+        second.setAttribute("dev", "hd" if cd_first else "cdrom")
         os_element.appendChild(second)
         # Apply new machine xml
         dom = self.libvirt_connection.defineXML(xml.toprettyxml())
         if dom is None:
-            raise Exception(f"Failed to set boot order cdrom first: {cd_first}, "
-                            f"for node: {node_name}")
-        logging.info(f"Boot order set successfully: cdrom first: {cd_first}, "
-                     f"for node: {node_name}")
+            raise Exception(f"Failed to set boot order cdrom first: {cd_first}, " f"for node: {node_name}")
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py:556]: merge the strings into one

File: discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py [hunk 4]
```diff
@@ -533,35 +537,32 @@ def set_per_device_boot_order(self, node_name, key: Callable[[Disk], int]):
...
+        logging.info(f"Boot order set successfully: cdrom first: {cd_first}, " f"for node: {node_name}")
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py:557]: merge the strings into one

File: discovery-infra/test_infra/controllers/node_controllers/ssh.py [hunk 5]
```diff
@@ -80,25 +83,27 @@ def script(self, bash_script, verbose=True, timeout=60):
             logging.info("Executing %s on %s", bash_script, self._ip)
             return self.execute(bash_script, timeout, verbose)
         except RuntimeError as e:
-            e.args += ('When running bash script "%s"' % bash_script),
+            e.args += (('When running bash script "%s"' % bash_script),)
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/node_controllers/ssh.py:86]:
```suggestion
            e.args += (f'When running bash script "{bash_script}"',)
```

File: discovery-infra/test_infra/controllers/node_controllers/ssh.py [hunk 6]
```diff
@@ -80,25 +83,27 @@ def script(self, bash_script, verbose=True, timeout=60):
             logging.info("Executing %s on %s", bash_script, self._ip)
             return self.execute(bash_script, timeout, verbose)
         except RuntimeError as e:
-            e.args += ('When running bash script "%s"' % bash_script),
+            e.args += ((f'When running bash script "{bash_script}"'),)
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/node_controllers/ssh.py:86]: you don't need the redundant inner parenthesis

**Review by osherdp — CHANGES_REQUESTED** [review]

File: discovery-infra/test_infra/assisted_service_api.py (hunk 1 above)
Comment (eliorerz) [reply discovery-infra/test_infra/assisted_service_api.py:183]: We are not using it.. I think we can just delete it

File: discovery-infra/test_infra/controllers/nat_controller.py [hunk 7]
```diff
@@ -12,13 +12,14 @@ class NatController:
...
     """
+
     def __init__(self, input_interfaces: Union[List, Tuple], ns_index: Union[str, int]):
         self._input_interfaces = input_interfaces
         self._ns_index = ns_index
         self._mark = self._build_mark()
 
     def add_nat_rules(self) -> None:
-        """" Add rules for the input interfaces and output interfaces """
+        """ Add rules for the input interfaces and output interfaces """
```
Comment (osherdp) [inline discovery-infra/test_infra/controllers/nat_controller.py:22]: remove whitespaces

**Pull request merged**